modules/virt-creating-a-vm-from-a-boot-source.adoc

```diff
@@ -1,28 +1,28 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-creating-and-using-default-os-images.adoc
+// * virt/virtual_machines/virtual_disks/virt-creating-and-using-boot-sources.adoc
 
-[id="virt-creating-a-vm-from-a-default-os-image_{context}"]
-= Creating a virtual machine from a default OS image
+[id="virt-creating-a-vm-from-a-boot-source_{context}"]
+= Creating a virtual machine from a boot source
 
-After you have created a persistent volume claim (PVC) for default operating system (OS) images, you create a new virtual machine or virtual machine template from a default OS image that you uploaded to the PVC.
+After you upload a boot source, you create a new virtual machine or virtual machine template from the boot source.
```

Comment:
This might beyond the scope of this PR, the section title is `Creating a virtual machine from a boot source`, so it's better to remove creation about `virtual machine template` in this section.